src/components/EditListingWizard/EditListingWizard.js

```diff
@@ -82,18 +89,33 @@ const tabLabel = (intl, tab) => {
  * @return true if tab / step is completed.
  */
 const tabCompleted = (tab, listing) => {
-  const { availabilityPlan, description, price, title, publicData } = listing.attributes
+  const { availabilityPlan, price, title, publicData } = listing.attributes // where does this come from?
```

Comment:
Not yet, however, I didn't let it block me, as all custom forms data must be stuffed in publicData, since only one of our forms is an actual sharetribe form, all of our listing data can be found inside of publicData.